Public/Src/Deployment/privatePackages.dsc

```diff
@@ -121,6 +121,12 @@ namespace PrivatePackages {
                         importFrom("BuildXL.Utilities.Instrumentation").withQualifier(winx64Qualifier).Common.dll,
                     ],
                 },
+                {
+                    subfolder: r`runtimes/win-x64/native/`,
+                    contents: [
+                        ...importFrom("BuildXL.Utilities").withQualifier(winx64Qualifier).Native.nativeWin,

```

Comment:
You probably don't need `.withQualifier(...)` because the value of `nativeWin` does not depend on it
```suggestion
                        ...importFrom("BuildXL.Utilities").Native.nativeWin,
```